### PR TITLE
Add the debug option and verbose (foreground)

### LINKED ISSUE #1954: Add the debug option and verbose (foreground)

### PR DESCRIPTION
**Pull request by folker-kuhn**
Fixes #1954

I am proposing this PR, so we can debug and do foreground run (or start?) easily. I took "suspend" and "verbose" names from some other projects, so I think the user is familiar with it.

### CODE REVIEW

**Comment by Thihup** [comment]
Hi @folker-kuhn.

As this is your first PR can you please acknowledge that the code will be owned by Manorrock.com. I will only ask this one time, as that is all that is needed for contributing code back to us.

About running in foreground, the `run.sh` does exactly that, so that it wouldn't be needed.
Also, could you implement the suspend part it in the `run.sh` script too?

**Comment by folker-kuhn** [comment]
I acknowledge the code in this PR and all further PRs I do to this project will be owned by Manorrock.com. Thanks!

Okay, that is what `run.sh` is for. I have to say that having both `start` and `run` is somewhat confusing. If you must have separate scripts for this, maybe `start-bg` and `start-fg` would be easier to grasp.

I'll look at `run.sh`

**Comment by mnriem** [comment]
@folker-kuhn Thanks for the acknowledgement. I'll let @Thihup work with you on the PR as he already engaged. We can now start merging any of your contributions. Welcome and thank you!

**Review by arjantijms — APPROVED** [review]
LGTM

File: server/src/main/zip/bin/run.sh [hunk 1]
```diff
@@ -1,23 +1,3 @@
...
-else
-    INIT_OPTIONS="-jar lib/piranha-server.jar"
-fi
-
-exec ${JAVA_BIN} ${JAVA_ARGS} -Djava.util.logging.config.file=etc/logging.properties ${INIT_OPTIONS} &
-
-PID=$!
-echo $PID >> tmp/piranha.pid
-wait $PID
+exec ./start.sh --run
```
Comment (Thihup) [inline server/src/main/zip/bin/run.sh]:
```suggestion
exec ./start.sh --run $*
```
Adding the `$*` allows to `./run.sh --suspend`

Comment (folker-kuhn) [reply server/src/main/zip/bin/run.sh]: I like that idea, thanks!

**Comment by folker-kuhn** [comment]
@Thihup what happened with the last commit and test run?

**Comment by folker-kuhn** [comment]
Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.3.0:jar (attach-javadocs) on project piranha-pico-core: MavenReportException: Error while generating Javadoc: 
Error:  Exit code: 1 - Loading source file /home/runner/work/piranha/piranha/pico/core/src/main/java/module-info.java...
Error:  Constructing Javadoc information...
Error:  error: No public or protected classes found to document.
Error:  1 error

I don't think this is related to my commit?

**Comment by Thihup** [comment]
@folker-kuhn It looks like @mnriem added a new subproject. However, it does not have any classes yet. So the Javadoc is complaining about it. It is not related to your changes. Sorry for the confusion.

**Comment by mnriem** [comment]
@folker-kuhn Thank you for your first contribution! It is MUCH appreciated and sorry for the confusion with the build ;)

**Pull request merged**